projection/ibc_channel_message/view/ibc_channel_messages_total.go

```diff
@@ -0,0 +1,16 @@
+package view
+
+import (
+	"github.com/crypto-com/chain-indexing/appinterface/projection/view"
+	"github.com/crypto-com/chain-indexing/appinterface/rdb"
+)
+
+type IBCChannelMessagesTotal struct {
+	*view.Total
+}
+
+func NewIBCChannelMessagesTotal(rdbHandle *rdb.Handle) *IBCChannelMessagesTotal {
+	return &IBCChannelMessagesTotal{
+		view.NewTotal(rdbHandle, "view_ibc_channel_messages_total"),
```

Comment:
Do we have an endpoint to get this data?

Reply:
Nope, It is mainly for pagination purpose. Not for any endpoint to direct query it.

Shamelessly copied from other projection 😂 . For example, `accountTransactions` has a similar view...

Reply:
Thanks, got it!